tests/PersonBearbeiten.spec.ts

```diff
@@ -195,4 +195,188 @@ test.describe(`Testfälle für die Administration von Personen": Umgebung: ${pro
             await personDetailsView.checkLockDateTo(sperrDatumBis); // Der Benutzer muss gesperrt sein bis zun dem eingegebenen Datum
         })
     })
+
+    test("Gesamtübersicht für einen Benutzer als Schueler öffnen und Unsichtbarkeit des 2FA Abschnitts prüfen", {tag: [LONG, STAGE]}, async ({ page }) => {
+        const vorname = "TAuto-PW-V-" + faker.person.firstName();
+        const nachname = "TAuto-PW-N-" + faker.person.lastName();
+        let userInfoLehrer: UserInfo;
+
+        await test.step(`Testdaten: Lehrer mit einer Rolle(LERN) über die api anlegen ${ADMIN}`, async () => {
+            userInfoLehrer = await createRolleAndPersonWithUserContext(page, testschule, 'LERN', nachname, vorname, await getSPId(page, 'itslearning'), await generateRolleName());
+            username.push(userInfoLehrer.username);
+            rolleId.push(userInfoLehrer.rolleId);
+        })
+
+        const personManagementView: PersonManagementViewPage = new PersonManagementViewPage(page);
+
+        const personDetailsView: PersonDetailsViewPage = await test.step(`Gesamtübersicht öffnen`, async () => {
+            await gotoTargetURL(page, "admin/personen"); 
+            await personManagementView.searchBySuchfeld(userInfoLehrer.username);
+            return await personManagementView.openGesamtuebersichtPerson(page, userInfoLehrer.username);
+        })
+
+        await test.step(`Unsichtbarkeit des 2FA Abschnitts prüfen`, async () => {
```

Comment:
Wenn z.B. der Klick für das öffnen der Gesamtübersicht nicht geklappt hat, merken die negativ-expects in Zeile 219-223 dieses nicht und der Test ist grün. 
Hier fehlt ein positiv assert auf die Gesamtübersicht.

Reply:
Das stimmt!